compiler/qsc_project/src/project.rs

```diff
@@ -36,6 +43,7 @@ pub struct Project {
     pub lints: Vec<LintOrGroupConfig>,
     /// Any errors encountered while loading the project.
     pub errors: Vec<Error>,
+    pub project_type: ProjectType,
```

Comment:
Can you add a doc comment? And maybe move this to under `path` to make the order more logical